### ISSUE TITLE
MQTT Passwort Länge eingeschränkt

### ISSUE THREAD

**Issue by ChrisKoh83** [Phantomias2006/Nemesis#112]
Mein gewähltes MQTT Passwort passt nicht in das Eingabefeld.

Habt ihr da ne Limitierung drin?

**Comment by sochs** [comment]
Ja, da ist eine Limitierung von 26 Zeichen drin. Der Speicher vom kleinen ESP gibt nicht mehr her.
https://github.com/WLANThermo-nano/WLANThermo_ESP82XX_Software/blob/master/data_source/data/index.html#L2470
